setup.py

```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="sanpy",
-    version="0.0.20",
+    version="0.1.20",
```

Comment:
If you bump the second number you need to set the last to 0 -> `0.1.0`

Reply:
ups yes, indeed